src/components/ha-cover-controls.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+
+import { UNAVAILABLE } from "../data/entity";
+import { HomeAssistant } from "../types";
+import { HassEntity } from "home-assistant-js-websocket";
+import CoverEntity from "../util/cover-model";
+
+import "./ha-icon-button";
+
+@customElement("ha-cover-controls")
+class HaCoverControls extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public stateObj!: HassEntity;
+
+  protected render() {
+    const entityObj = new CoverEntity(this.hass, this.stateObj);
+
+    return html`
+      <div class="state">
+        <ha-icon-button
+          class=${classMap({
+            invisible: entityObj.supportsOpen,
```

Comment:
Can't we just not render the icon button at all?

```ts
${entityObj.supportsOpen ? html`` : ""}
```

Reply:
want to maintain spacing